Copyright notice for
tutorials/far/CMakeLists.txt:
tutorials/hbr/tutorial_0/CMakeLists.txt:
tutorials/hbr/tutorial_1/CMakeLists.txt:
tutorials/hbr/tutorial_2/CMakeLists.txt:
tutorials/osd/tutorial_0/CMakeLists.txt:

#
#   Copyright 2013 Pixar
#
#   Licensed under the Apache License, Version 2.0 (the "Apache License")
#   with the following modification; you may not use this file except in
#   compliance with the Apache License and the following modification to it:
#   Section 6. Trademarks. is deleted and replaced with:
#
#   6. Trademarks. This License does not grant permission to use the trade
#      names, trademarks, service marks, or product names of the Licensor
#      and its affiliates, except as required to comply with Section 4(c) of
#      the License and to reproduce the content of the NOTICE file.
#
#   You may obtain a copy of the Apache License at
#
#       http://www.apache.org/licenses/LICENSE-2.0
#
#   Unless required by applicable law or agreed to in writing, software
#   distributed under the Apache License with the above modification is
#   distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
#   KIND, either express or implied. See the Apache License for the specific
#   language governing permissions and limitations under the Apache License.
#

# This patch was obtained from the gentoo-overlay.

diff -Naur a/tutorials/far/CMakeLists.txt b/tutorials/far/CMakeLists.txt
--- a/tutorials/far/CMakeLists.txt	2020-05-11 14:48:35.145797436 +1000
+++ b/tutorials/far/CMakeLists.txt	2020-05-11 14:49:39.445969003 +1000
@@ -30,7 +30,7 @@
         $<TARGET_OBJECTS:far_obj>
      )
 
-     install(TARGETS ${NAME} DESTINATION "${CMAKE_BINDIR_BASE}/tutorials")
+     install(TARGETS ${NAME} DESTINATION "${CMAKE_BINDIR_BASE}")
 
 endmacro()
 
diff -Naur a/tutorials/hbr/tutorial_0/CMakeLists.txt b/tutorials/hbr/tutorial_0/CMakeLists.txt
--- a/tutorials/hbr/tutorial_0/CMakeLists.txt	2020-05-11 14:48:35.145797436 +1000
+++ b/tutorials/hbr/tutorial_0/CMakeLists.txt	2020-05-11 14:50:38.076125987 +1000
@@ -30,5 +30,5 @@
     ${SOURCE_FILES}
 )
 
-install(TARGETS hbr_tutorial_0 DESTINATION "${CMAKE_BINDIR_BASE}/tutorials")
+install(TARGETS hbr_tutorial_0 DESTINATION "${CMAKE_BINDIR_BASE}")
 
diff -Naur a/tutorials/hbr/tutorial_1/CMakeLists.txt b/tutorials/hbr/tutorial_1/CMakeLists.txt
--- a/tutorials/hbr/tutorial_1/CMakeLists.txt	2020-05-11 14:48:35.145797436 +1000
+++ b/tutorials/hbr/tutorial_1/CMakeLists.txt	2020-05-11 14:50:10.406051839 +1000
@@ -30,5 +30,5 @@
     ${SOURCE_FILES}
 )
 
-install(TARGETS hbr_tutorial_1 DESTINATION "${CMAKE_BINDIR_BASE}/tutorials")
+install(TARGETS hbr_tutorial_1 DESTINATION "${CMAKE_BINDIR_BASE}")
 
diff -Naur a/tutorials/hbr/tutorial_2/CMakeLists.txt b/tutorials/hbr/tutorial_2/CMakeLists.txt
--- a/tutorials/hbr/tutorial_2/CMakeLists.txt	2020-05-11 14:48:35.145797436 +1000
+++ b/tutorials/hbr/tutorial_2/CMakeLists.txt	2020-05-11 14:50:20.706079428 +1000
@@ -30,5 +30,5 @@
     ${SOURCE_FILES}
 )
 
-install(TARGETS hbr_tutorial_2 DESTINATION "${CMAKE_BINDIR_BASE}/tutorials")
+install(TARGETS hbr_tutorial_2 DESTINATION "${CMAKE_BINDIR_BASE}")
 
diff -Naur a/tutorials/osd/tutorial_0/CMakeLists.txt b/tutorials/osd/tutorial_0/CMakeLists.txt
--- a/tutorials/osd/tutorial_0/CMakeLists.txt	2020-05-11 14:48:35.145797436 +1000
+++ b/tutorials/osd/tutorial_0/CMakeLists.txt	2020-05-11 14:50:51.446161856 +1000
@@ -35,5 +35,5 @@
     osd_static_cpu
 )
 
-install(TARGETS osd_tutorial_0 DESTINATION "${CMAKE_BINDIR_BASE}/tutorials")
+install(TARGETS osd_tutorial_0 DESTINATION "${CMAKE_BINDIR_BASE}")
 
